src/main/java/io/github/fablabsmc/fablabs/impl/fiber/tree/ConfigNodeImpl.java

```diff
@@ -82,7 +82,9 @@ public ConfigBranch getParent() {
 
 		if (attr != null) {
 			checkAttributeType(expectedType, attr);
-			return Optional.ofNullable(expectedType.getPlatformType().cast(attr.getValue()));
+			@SuppressWarnings("unchecked")
+			A a = (A) expectedType.getErasedPlatformType().cast(attr.getValue());
```

Comment:
Just a thought, we may be able to add a `platformCast` method directly to `SerializableType`, avoiding a lot of unchecked casts like that.